### PR TITLE
[Maintenance] Configure builds on postgressql & mariadb

### PR DESCRIPTION
**Pull request by Arminek**
| Q               | A
| --------------- | -----
| Branch?         | master 
| Bugfix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | -
| License         | MIT

🖖 Hello, Sylius community,
We want to be better at supporting DB like Postgresql & MariaDB. So our first step in this direction is running our test suite on these DBs. To see what is not working correctly. Next steps -> fix these errors one by one. Stay tuned! 🖖

### CODE REVIEW

**Review by arti0090 — APPROVED** [review]

File: .github/workflows/application.yml [hunk 1]
```diff
@@ -109,7 +109,213 @@ jobs:
                 run: vendor/bin/phpstan analyse
                 if: always() && steps.end-of-setup.outcome == 'success'
 
-    test-application-without-frontend:
+    test-application-without-frontend-postgres:
+        runs-on: ubuntu-18.04
+
+        name: "Test non-JS application (PHP ${{ matrix.php }}, Symfony ${{ matrix.symfony }}, PostgresSql ${{ matrix.postgres }})"
```
Comment (GSadee) [inline .github/workflows/application.yml:115]:
```suggestion
        name: "Test non-JS application (PHP ${{ matrix.php }}, Symfony ${{ matrix.symfony }}, PostgreSQL ${{ matrix.postgres }})"
```
?

**Review by lchrusciel — APPROVED** [review]

**Comment by lchrusciel** [comment]
The base of this pull-request was changed, you need fetch and reset your local branch 
if you want to add new commits to this pull request. **Reset before you pull, else commits
may become messed-up.**

Unless you added new commits (to this branch) locally that you did not push yet, 
execute `git fetch origin && git reset "maintenance/postgress-builds"` to update your local branch.
 
Feel free to ask for assistance when you get stuck :+1:

**Comment by lchrusciel** [comment]
The base of this pull-request was changed, you need fetch and reset your local branch 
if you want to add new commits to this pull request. **Reset before you pull, else commits
may become messed-up.**

Unless you added new commits (to this branch) locally that you did not push yet, 
execute `git fetch origin && git reset "maintenance/postgress-builds"` to update your local branch.
 
Feel free to ask for assistance when you get stuck :+1:

**Review by Zales0123 — APPROVED** [review]
Maybe it would be worth extracting these configurations to the separate workflow file? Just to keep things cleaner :)

File: .github/workflows/application.yml [hunk 2]
```diff
@@ -109,7 +109,214 @@ jobs:
                 run: vendor/bin/phpstan analyse
                 if: always() && steps.end-of-setup.outcome == 'success'
 
-    test-application-without-frontend:
+    test-application-without-frontend-postgres:
+        runs-on: ubuntu-18.04
+
+        name: "Test non-JS application (PHP ${{ matrix.php }}, Symfony ${{ matrix.symfony }}, PostgresSQL ${{ matrix.postgres }})"
```
Comment (GSadee) [inline .github/workflows/application.yml:115]:
```suggestion
        name: "Test non-JS application (PHP ${{ matrix.php }}, Symfony ${{ matrix.symfony }}, PostgreSQL ${{ matrix.postgres }})"
```

**Review by GSadee — APPROVED** [review]

**Comment by GSadee** [comment]
Thank you, Arkadiusz! :1st_place_medal:

**Pull request merged**